src/CameraGstreamer.cpp

```diff
@@ -37,6 +37,7 @@ CameraGstreamer::CameraGstreamer( CameraId id,
         const DUCameraDescriptor description,
         CamerasManager *manager,
         unsigned framesPerSecond ):
+    _bufferSize( getBufferSize( description.format, description.width*description.height ) ),
```

Comment:
Not relevant for this fix, but getBufferSize must be a part of our API in one way or another.

Reply:
Agreed.
The best I can do for now is open a ticket:
https://drive-u.atlassian.net/browse/MIYAD-3005